### PR TITLE
Solve small issues

### PR DESCRIPTION
**Pull request by ricardoavelino**
This PR reflects:
- some problems noted in the usage of v.1.0.0 after working with the students.
- some new cmds to do things in mac that can only be done via the displaysetting tables (autoupdate on/off and deviation limit)

Bye

### CODE REVIEW

File: ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import scriptcontext as sc
+
+import compas_rhino
+
+
+__commandname__ = "AGS_form_deviation_threshold"
+
+
+def RunCommand(is_interactive):
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py:13]: why is this not just part of the global settings dialog?

File: ui/Rhino/AGS/dev/AGS_form_from_layer_cmd.py [hunk 2]
```diff
@@ -20,10 +20,12 @@ def RunCommand(is_interactive):
 
     scene = sc.sticky['AGS']['scene']
 
-    layer = compas_rhino.rs.CurrentLayer()
-    layer_name = compas_rhino.rs.GetString("Layer to construct FormDiagram", layer)
+    layer_name = compas_rhino.rs.GetString("Type name of layer  with lines to construct FormDiagram")
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_form_from_layer_cmd.py:23]: to be honest i would just call this `layer`. less underscores make the code more readable...

File: ui/Rhino/AGS/dev/AGS_form_update_from_force_cmd.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 import scriptcontext as sc
 
 import compas_rhino
+import AGS_form_check_deviation_cmd
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_form_update_from_force_cmd.py:8]: same here

File: ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+
+__commandname__ = "AGS_toggle_autoupdate"
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py:10]: the name of the file has a typo

File: ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+def RunCommand(is_interactive):
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py:13]: as with the maxdev, i think this should be part of the general settings dialog...

File: ui/Rhino/AGS/dev/AGS_edge_information_cmd.py [hunk 6]
```diff
@@ -79,7 +79,8 @@ def RunCommand(is_interactive):
         find_object(key2guid[edge_form]).Select(True)
         key2guid = {force.guid_edge[guid]: guid for guid in force.guid_edge}
         key2guid.update({(v, u): key2guid[(u, v)] for u, v in key2guid})
-        find_object(key2guid[edge_force]).Select(True)
+        if abs(f) > 10e-4:
```
[the remark this answers is not in the document]
Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_edge_information_cmd.py:82]: Ok, indeed I believe that this tolerance already exists. I will just update to refer to it.

File: ui/Rhino/AGS/dev/AGS_force_move_nodes_cmd.py [hunk 7]
```diff
@@ -5,6 +5,7 @@
 import scriptcontext as sc
 
 import compas_rhino
+import AGS_form_check_deviation_cmd
```
[the remark this answers is not in the document]
Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_force_move_nodes_cmd.py:8]: Ok, I actually thought that this was desirable to centralise the messages. Will update!

File: ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py (hunk 1 above)
Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py:13]: Hi tom. This is part of the general dialog! Therefore there's no way how this is editable via mac. So this function and the toggle_autoupdate are there to bring it to mac. You have other suggestion?

File: ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py (hunk 4 above)
Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py:10]: Ok

File: ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py (hunk 5 above)
Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py:13]: This was again created to compatibility with mac

File: ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py (hunk 1 above)
Comment (brgcode) [reply ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py:13]: there should be a general settings form for Mac as well. RV2 has one...

Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py:13]: ok, so should I remove this from the PR, and open an issue in the issue tracker?

Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py:13]: Or you think we can accept it and open the issue to deal with it in the future? I would say that it is annoying currently not being able to change these values from AGS... does this pass as a temporary solution?

File: ui/Rhino/AGS/dev/AGS_force_move_nodes_cmd.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (brgcode) [reply ui/Rhino/AGS/dev/AGS_force_move_nodes_cmd.py:8]: well we could indeed centralise messages but that would be a different thing...

**Pull request merged**